code/modules/modular_computers/hardware/identifier.dm

```diff
@@ -10,4 +10,7 @@
 	var/name = holder.saved_identification
 	var/job = holder.saved_job
 
-	holder.name = "PDA-[name] ([job])"
+	if(istype(holder, /obj/item/modular_computer/tablet/pda))
+		holder.name = "PDA-[name] ([job])"
+	else if(istype(holder, /obj/item/modular_computer/tablet))
+		holder.name = "Tablet-[name] ([job])"
```

Comment:
Identifier modules work because you are supposed to be able to display your PDA on your belt. Can tablets *really* fit on a belt? I don't know if we want to make this a thing.

We should not remove the unique elements of PDAs, that being, they are messengers and ID stores. Giving all this function to tablets sort of removes ANY benefit from using a PDA at all.

Reply:
It goes against my idea that PDAs should have benefits over Tablets.

If a player is going to engage with the system they should be rewarded.

There are no pre-made tablets. I think this is fitting.

And yes, they can be used in belt slots already (not something this PR touched)